### PR TITLE
feat(operator): add combineLatest operator

### PR DESCRIPTION
**Pull request by raininfall**
add combineLatest operator

solve #26

### CODE REVIEW

File: fx/fx.go [hunk 1]
```diff
@@ -16,7 +16,10 @@ type (
 
 	// FilterableFunc defines a func that should be passed to the Filter operator.
 	FilterableFunc func(interface{}) bool
-
```
Comment (jochasinga) [inline fx/fx.go]: Omit extra line here.

File: observable/observable.go [hunk 2]
```diff
@@ -348,3 +350,39 @@ func Merge(o1 Observable, o2 Observable, on ...Observable) Observable {
 	}()
 	return Observable(out)
 }
+
+//CombineLatest emits an item whenever any of the source Observables emits an item
```
Comment (jochasinga) [inline observable/observable.go]: Single space after `//`

File: fx/fx.go (hunk 1 above)
Comment (raininfall) [reply fx/fx.go]: I don't know why it shows like this.
There is extra line in source file.

File: observable/observable.go (hunk 2 above)
Comment (raininfall) [reply observable/observable.go]: Fixed.

File: fx/fx.go (hunk 1 above)
Comment (raininfall) [reply fx/fx.go]: It just delete some white space and add an extra line here.
https://github.com/ReactiveX/RxGo/pull/40/files/1bba7e0e83de197091dfb0ece99e453aabacc356

**Pull request merged**